content/support/compressed-fastq/index.md

```diff
@@ -22,31 +22,11 @@ Uploaded gz compressed FASTQ data loads in compressed format directly into the H
 Why bother? Compressed data saves space in your account. This is a priority for many that have larger sized data/experiments to analyze. As before, some tools accept fastq datatypes (example: prep/QA steps/tools) and others accept fastqsanger datatypes (example: mapping and downstream analysis steps/tools). See the tool form to know which is expected. **When in doubt, use fastqsanger**.
 
 
-### Using compressed data as tool inputs
+[Using compressed data as tool inputs](https://training.galaxyproject.org/training-material/faqs/galaxy/datatypes_Using_compressed_fastq_data.html)
```

Comment:
I think that this may break the anchor link in the markdown title, so I'd suggest to keep the "###"
 
```suggestion
### [Using compressed data as tool inputs](https://training.galaxyproject.org/training-material/faqs/galaxy/datatypes_Using_compressed_fastq_data.html)
```

Reply:
Should I keep the headers for all of the replaced FAQs?

Reply:
I think so! That way we ensure that links won't be broken

Reply:
Okay I will add the headers as well. Thanks!

Reply:
thanks to you :)